pkg/controller/plan/migration.go

```diff
@@ -415,6 +443,14 @@ func (r *Migration) CleanUp(vm *plan.VMStatus) (err error) {
 	if err != nil {
 		return
 	}
+	err = r.kubevirt.DeletePopulatorDataSource(vm)
+	if err != nil {
+		return
+	}
+	err = r.kubevirt.DeletePopulatorPods(vm)
```

Comment:
it doesn't matter that this code is called from three different places if we always check that the migration succeeded here before removing the importer pods.. I mentioned this because it could make sense to follow the same pattern for the populator pods